js/src/components/google-ads-account-card/connect-ads/ads-account-select-control.js

```diff
@@ -1,13 +1,12 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import AppSelectControl from '.~/components/app-select-control';
-import toAccountText from '.~/utils/toAccountText';
```

Comment:
As far as I can see the function toAccountText is no longer used after this removal, should we remove that from the utils directory as well?